apps/desktop-wallet/locales/bg-BG/translation.json

```diff
@@ -462,5 +462,10 @@
   "walletConnectProposalMessage": "Connect to <1>{{ dAppUrl }}</1> with one of your addresses:",
   "walletConnectProposalMessageWithGroup": "Connect to <1>{{ dAppUrl }}</1> with one of your addresses in group <1>{{ group }}</1>:",
   "Active dApp connections": "Active dApp connections",
-  "Connect to {{ dAppUrl }}": "Connect to {{ dAppUrl }}"
+  "Connect to {{ dAppUrl }}": "Connect to {{ dAppUrl }}",
+  "The address which will receive the transaction.": "The address which will receive the transaction.",
+  "Select the address to send funds to.": "Select the address to send funds to.",
```

Comment:
Hm, is the address receiving the transaction, or the tokens/funds? Do we ever say "Did you receive the transaction I sent you?"? :/ I think we more often say "Did you receive the coins/tokens/funds/assets I sent you?"

So here, we are using both. In 466 we say "transaction" and below "funds". I personally would go for "funds".

WDYT?

Reply:
I think "funds" is better, because "asset" represents the whole currency IMO. So, when sending 5 ALPH, the asset is ALPH and the funds are 5 ALPH. What do you think about this distinction? 

Besides, "Funds are SAFU" 😆